Components/grade-table.js

```diff
@@ -1,24 +1,49 @@
 class GradeTable {
-  constructor(tableElement) {
+  constructor(tableElement, noGradesElement) {
     this.tableElement = tableElement;
+    this.noGradesElement = noGradesElement;
   }
 
   updateGrades(grades) {
     console.log(grades);
     var tBody = $("#table");
     tBody.empty();
```

Comment:
Use the saved `tableElement` property instead of re-querying the entire `document`.